### PR TITLE
Ticket 1069 - mobile styling

### LINKED ISSUE #1069: Mobile - Align UI elements like in PROD
1. Map widgets aligned to the bottom-right
2. Left panel tabs go to the bottom of the screen
3. Footer goes just above the left panel tabs
Note -- Let's not worry about that MORE button or header in this PR.

Map widgets should go to the bottom-right corner:
Screen Shot 2020-05-27 at 1 03 20 PM

Tabview  should be going to the very bottom of the screen:
Screen Shot 2020-05-27 at 1 05 14 PM

PROD Example:
image

### PR DESCRIPTION
**Pull request by eyosef**
This PR implements mobile styling to align the app with prod. Fixes #1069

### CODE REVIEW

File: src/css/mapWidgets.scss [hunk 1]
```diff
@@ -36,7 +36,14 @@
 
 @media screen and (max-width: 475px) {
   .widget-wrapper {
-    right: 0px;
+    // top: auto;
+    // right: auto;
+    // left: 19rem;
+    // bottom: 7rem;
```
Comment (vaidotasp) [inline src/css/mapWidgets.scss]: let's get rid of those commented out lines?

File: src/css/mapWidgets.scss [hunk 2]
```diff
@@ -36,7 +36,14 @@
...
+    top: auto;
+    right: 0;
+    /* left: 19rem; */
```
Comment (vaidotasp) [inline src/css/mapWidgets.scss]: same here

**Review by vaidotasp — CHANGES_REQUESTED** [review]
Screen Shot 2020-05-30 at 11 29 39 AM
Can we somehow get rid of that second scroll whitespace? it is happening due to overflow scroll on the left panel.

**Comment by eyosef** [comment]
@vaidotasp I couldn't reproduce the `overflow` style bug you ID'd, but I moved my use of `overflow` in `.tabview-container` into a media query since it's only needed in that context. Let me know what you think!

**Comment by vaidotasp** [comment]
looks good

**Pull request merged**